ci/3-setup-k8s

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# NOTE: The script should be executed as a part of ci/run!
+
+# SCRIPT SUMMARY
+# --------------
+# 1. Start and await minikube, if required
+# 2. Initialize Helm, if required
+
+# FIXME: currently we only check if minikube+helm is ready, not first if
+#        minikube is ready and then if also helm is ready.
+
+# Set shell options
+# -e : Exit immediately if a command exits with a non-zero status.
+# -u : Treat unset variables as an error when substituting.
+# -x : Print commands and their arguments as they are executed.
+set -eux
+
+# Setup Kubernetes
+# ----------------
+
+# Don't use the default profile: this helps to make it clear whats what while
+# inspecting virtualbox's VMs.
+minikube profile z2jh
+
+if set +u && [[ ! -z "${CICD}" ]] && set -u; then
+  # CICD
+  echo "starting minikube - CICD mode"
+  minikube config set vm-driver none
+  minikube config set WantNoneDriverWarning false
+  export CHANGE_MINIKUBE_NONE_USER=true
+
+  minikube addons disable dashboard
+  SUDO='sudo -E'
+else
+  # LOCAL
+  if (timeout 3 bash -c 'kubectl get pod -n kube-system --selector app=helm,name=tiller | grep 1/1'); then
+    echo "minikube cluster was already setup"
```

Comment:
I'm confused by this. The message makes me think that this bit of code had tried to setup the minikube cluster but then decided it didn't have to. However there is no code here that sets up a cluster.

Just a language thing? "minikube cluster configured and up and running"?

Reply:
Ah lets make the language change!